Example/src/main/java/com/parse/example/Room.java

```diff
@@ -0,0 +1,27 @@
+package com.parse.example;
+
+import com.parse.ParseClassName;
+import com.parse.ParseObject;
+
+@ParseClassName("Room")
+public class Room extends ParseObject {
+
+    public Room() {
+
+    }
+
+    String name;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
```

Comment:
ParseObject using key-value pair to manage data.
You need to use [put(key, value)](https://github.com/ParsePlatform/Parse-SDK-Android/blob/05795898c2cf63e78bedba6c9e20f569294685df/Parse/src/main/java/com/parse/ParseObject.java#L158)/[get(key)](https://github.com/ParsePlatform/Parse-SDK-Android/blob/05795898c2cf63e78bedba6c9e20f569294685df/Parse/src/main/java/com/parse/ParseObject.java#L290) inside your getter/setter. (That's why you can't retrieve data from the server.)

ex.
```java
@ParseClassName("Room")
public class Room extends ParseObject {
  
  public Room() {

  }

  public String getName() {
    return getString("name");
  }

  public void setName(String name) {
    put("name", name);
  }

}
```